pkg/mounter/ossfs.go

```diff
@@ -0,0 +1,265 @@
+package mounter
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/kubernetes-sigs/alibaba-cloud-csi-driver/pkg/utils"
+	log "github.com/sirupsen/logrus"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/apimachinery/pkg/api/resource"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+	"k8s.io/client-go/kubernetes"
+	mountutils "k8s.io/mount-utils"
+	"k8s.io/utils/pointer"
+)
+
+var defaultOssfsImageTag = "81b7e56-aliyun"
+
+const (
+	hostPrefix                = "/host"
+	OssfsCredentialSecretName = "csi-ossfs-credentials"
+	OssfsDefMimeTypesFilePath = "/etc/mime.types"
+	OssfsCsiMimeTypesFilePath = "/etc/csi-mime.types"
+)
+
+type fuseOssfs struct {
+	config FuseContainerConfig
+}
+
+func NewFuseOssfs(configmap *corev1.ConfigMap) FuseMounterType {
+	config := extractFuseContainerConfig(configmap, "ossfs")
+	// set default image
+	if config.Image == "" {
+		registry := os.Getenv("DEFAULT_REGISTRY")
+		if registry == "" {
+			registry = fmt.Sprintf("registry-vpc.%s.aliyuncs.com", utils.RetryGetMetaData("region-id"))
```

Comment:
```suggestion
			registry = fmt.Sprintf("registry-%s-vpc.ack.aliyuncs.com", utils.RetryGetMetaData("region-id"))
```

Reply:
or use "registry.cn-beijing.aliyuncs.com" instead

Reply:
> or use "registry.cn-beijing.aliyuncs.com" instead

Yes, this may be better. We have some regions that don't have dedicated ACR instances. And we also have non-ECS deployment.

Reply:
@AlbeeSo  Why we have "registry.cn-beijing.aliyuncs.com/acs/csi-ossfs", but not "registry.cn-beijing.aliyuncs.com/acs/csi-ossfs"? Or would "registry-cn-hangzhou.ack.aliyuncs.com" be even better?